cmd/create_config.go

```diff
@@ -38,16 +38,16 @@ func generateKeys(config config.Config) {
 	keytools.GenerateSigningKeyPair(ethAuthPub, ethAuthPvt, "secp256k1")
 }
 
-func addKeys() error {
-	err := identity.IDService.AddKeyFromConfig(identity.KeyPurposeP2P)
+func addKeys(idService identity.Service) error {
```

Comment:
Why not make the function part of `identity.Service`? Even `createIdentity` ?

Reply:
I dont want to do functional changes on this pr,

Reply:
this would surely come in a later pr.

Reply:
i take the above comment back, this is not part of business logic, its just a user utility doesn't make sense to be in a service.